Source/Models/BrickDimension.swift

```diff
@@ -18,6 +18,41 @@ public struct BrickSize {
     }
 }
 
+public enum RestrictedSize {
+    case minimumSize(size: BrickDimension)
+    case maximumSize(size: BrickDimension)
+}
+
+public struct RestrictedBrickSize {
+    let size: BrickDimension
+    let restrictedSize: RestrictedSize
+    
+    var isEstimate: Bool {
+        switch size {
+        case .auto(_): return true
+        default: return false
```

Comment:
Thats not really applicable, if you're not using auto there is no point of a restriction